activerecord/lib/active_record/asynchronous_queries_tracker.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "weakref"
+
+module ActiveRecord
+  class AsynchronousQueriesTracker # :nodoc:
+    class << self
+      def install_executor_hooks(executor = ActiveSupport::Executor)
+        executor.register_hook(self)
+      end
+
+      def run
+      end
+
+      def complete(_pools)
```

Comment:
https://api.rubyonrails.org/v6.0.3.2/classes/ActiveSupport/ExecutionWrapper.html#method-c-register_hook

> `hook.complete` will be passed the value returned from `hook.run`

The name `_pools` doesn't really fit here. Elsewhere this parameter is called `_state` when it's always ignored:

https://github.com/rails/rails/blob/35a103a96e4879df1b84062fa2ac6aa9d97ded71/railties/lib/rails/application/finisher.rb#L152

Reply:
Thanks! I'll correct that.